src/state/stake/hooks.ts

```diff
@@ -194,14 +194,13 @@ export function useSyrupInfo(
       chainId
         ? returnSyrupInfo()
             [chainId]?.slice(startIndex, endIndex)
-            .filter((stakingRewardInfo) =>
-              tokenToFilterBy === undefined || tokenToFilterBy === null
-                ? getSearchFiltered(
-                    stakingRewardInfo,
-                    filter ? filter.search : '',
-                  )
-                : tokenToFilterBy.equals(stakingRewardInfo.token) &&
-                  tokenToFilterBy.equals(stakingRewardInfo.token),
+            .filter(
+              (syrupInfo) =>
+                syrupInfo.ending > Math.floor(Date.now() / 1000) &&
+                (tokenToFilterBy === undefined || tokenToFilterBy === null
+                  ? getSearchFiltered(syrupInfo, filter ? filter.search : '')
+                  : tokenToFilterBy.equals(syrupInfo.token) &&
+                    tokenToFilterBy.equals(syrupInfo.token)),
```

Comment:
What is this checking?
tokenToFilterBy.equals(syrupInfo.token) &&
                    tokenToFilterBy.equals(syrupInfo.token)

Looks like the same condition